src/test/system_tests/test_bucket_access.js

```diff
@@ -11,17 +11,28 @@ var rpc = api.new_rpc();
 var argv = require('minimist')(process.argv);
 var assert = require('assert');
 var AWS = require('aws-sdk');
-var https = require('https');
+// var https = require('https');
 var fs = require('fs');
 var uuid = require('uuid/v4');
 
 
 dotenv.load();
 
 
+const {
+    no_setup,
+    mgmt_ip = '127.0.0.1',
+    mgmt_port = '8080',
+    s3_ip = '127.0.0.1',
+    s3_port = '80',
+} = argv;
+
+
+const target_s3_endpoint = `http://${s3_ip}:${s3_port}`;
+
 
 var client = rpc.new_client({
-    address: 'ws://127.0.0.1:' + process.env.PORT
+    address: `ws://${mgmt_ip}:${mgmt_port}`
```

Comment:
This is not the only test which uses process.env.port for RPC, better change all?

Reply:
I worked on this one as an example since it's very simple. next step is to go over all tests and fix them accordingly

Reply:
Resolving, but we need to remember this